addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('mail.notification_alert', function (require) {
+"use strict";
+
+var widgetRegistry = require('web.widget_registry');
+var Widget = require('web.Widget');
+
+var notificationAlert = Widget.extend({
+    renderElement: function () {
```

Comment:
You should use a template instead of overriding `renderElement`.

```js
var NotificationAlert = Widget.extend({
   template: 'mail.NotificationAlert',
   /**
    * @override
    */
   init: function () {
      this._super.apply(this, arguments);
      this.isNotificationDenied = window.Notification && windnow.Notification.permission === "denied":
   },
});
```

```xml
<!--
   @param {mail.NotificationAlert} widget
-->
<t t-name="mail.NotificationAlert">
   <span t-if="widget.isNotificationDenied" class="o_notification_alert alert alert-primary">
       Odoo Push notifications have been blocked. Go to your browser settings to allow them.
   </span>
</t>
```

This solution also automatically marks the text as translatable.